filters/magick.cpp

```diff
@@ -109,6 +109,7 @@ magick_version_before_(const char *magick, const char *cutoff)
         {
           log::debug ("found %1%-%2%") % magick % version;
           return (0 > strverscmp (version, cutoff));
+          return (false);
```

Comment:
This will never be reached but you already fixed that in a later commit.  In such a case you can rebase interactively and merge the two commits into one.  That way, I won't be scratching my head for a while :wink: